liberty-maven-plugin/src/it/tests/ear-project-it/SampleEAR/pom.xml

```diff
@@ -60,6 +66,10 @@
                             <artifactId>SampleWAR</artifactId>
                             <uri>/modules/web.war</uri>
                         </webModule>
+                        <webModule>
+                            <groupId>net.wasdev.wlp.maven.it</groupId>
+                            <artifactId>SampleWAR2</artifactId>
```

Comment:
Add a comment here that says:
`<!-- uri and bundleDir elements not included to test issue 264 -->`